CODE_OF_CONDUCT.md

```diff
@@ -6,14 +6,31 @@ identity and expression, sexual orientation, disability, personal
 appearance, body size, race, ethnicity, age, religion, nationality, or
 other such characteristics.
 
-Everyone is expected to follow the [Scala Code of Conduct] when
-discussing the project on the available communication channels. If you
-are being harassed, please contact us immediately so that we can
-support you.
+The ZIO project follows the [Scala Code of Conduct](https://www.scala-lang.org/conduct/), with
+adjustments with clarifications with regard to moderation that are detailed below.
+All participants, contributors and members are expected to follow the Scala Code of Conduct
+when discussing the project on the available communication channels. If you are being harassed, 
+please contact us immediately so that we can support you.
 
- ## Moderation
+## Moderation and Steering Committee
+ 
+The ZIO project's group of moderators is the Steering Committee team. The members currently are:
```

Comment:
I'd rephrase this as `The ZIO project is moderated by the Steering Committee team members:`.

Reply:
Need to add @ghostdogpr to this list

Reply:
and add @wi101